app/views/mentors/van_drivers.html.erb

```diff
@@ -25,6 +49,22 @@
 </table>
 
 <div id="sidebar">
-  <h3>Update Driver Status</h3>
-  <div id="form-container">Click a row to begin editing</div>
-</div>
\ No newline at end of file
+	<p><%= render :partial => "shared/term_select",
+				:locals => { :form_url => van_drivers_mentors_path },
+				:object => @term %></p>
+	<hr />
+	<h3>Update Driver Status</h3>
+	<div id="form-container">
+		<p>Click a row to begin editing</p>
+		<%- if Customer.link_to_uw? -%>
+		<p>--OR--</p>
+		<%= link_to "Check UWFS Training for #{@term.is_a?(Quarter) ? @term.title : @term.to_param}", driver_training_status_mentor_url(999), :class => 'sync button', :id => 'check_all_button' %>
+		<%- end -%>
+	</div>
+</div>
+
+<%= javascript_tag do %>
+	sidebar_content = $("#sidebar").html();
```

Comment:
I like the easy "sidebar editor" concept. Maybe we can bring that to other places in the UI.